BlueBackPixel.java

```diff
@@ -537,40 +539,28 @@ private void armextend(double extend_retract, double power) {
   // GRABBER COMMAND FUNCTIONS
   //**********************************************
   // Description: 1st function sets servo to run to position, then moves on
-  // 2nd function sets servo to run to position, waits until position is reached
-  // plus a little extra
-  // idea is to save time, only waiting as long as necessary as opposed to
-  // an arbitrary time, such as 100 ms. only thing needed is a "wait" string
-  // for function parameters
-  private void closeClamp() { closeClamp(GRABBER_SERVO_PAUSE_TIME_MS); }
-  private void closeClamp(int pause_time_ms) {
-    grabber.setPosition(GRABBER_SERVO_CLOSED_POS);
-    until (grabber.getPosition == GRABBER_SERVO_CLOSED_POS) {
-      sleep(1);
-    }
-    sleep(5);
+  // 2nd function finds how far the servo is moving, and uses the distance
+  // to calculate a more customized wait time, instead of an arbitrary number
+  private void closeClamp() { grabber.setPosition(GRABBER_SERVO_CLOSED_POS); }
+  private void closeClampWait() {
+    double distance = Math.abs(grabber.getPosition - GRABBER_SERVO_CLOSED_POS); // Calculate movement BEFORE setting new position
+    grabber.setPosition(GRABBER_SERVO_OPENED_POS); // Set new position
```

Comment:
We found and fixed this issue (_OPENED_ on line 547 should be _CLOSED_ like the rest of this function) at Tuesday's team meeting.